apps/wizdm/src/app/navigator/actionbar/action.component.ts

```diff
@@ -23,9 +23,14 @@ export class ActionComponent {
   @Input() color: ThemePalette;
 
   /** Disables the action */
-  @Input('disabled') set disableAction(value: boolean) { this.disabled = coerceBooleanProperty(value); }
+  @Input('disabled') set disableAction(value: any) { this.disabled = coerceBooleanProperty(value); }
```

Comment:
I changed type to any, as it better reflects what `coerceBooleanProperty` will do...